benchmark/fft/simple_fft_benchmark_reporter.cc

```diff
@@ -0,0 +1,25 @@
+#include "benchmark/fft/simple_fft_benchmark_reporter.h"
+
+#include <string>
+#include <vector>
+
+#include "tachyon/base/strings/string_number_conversions.h"
+
+namespace tachyon {
+
+SimpleFFTBenchmarkReporter::SimpleFFTBenchmarkReporter(
+    std::string_view title, const std::vector<uint64_t>& exponents)
+    : exponents_(exponents) {
+  title_ = std::string(title);
```

Comment:
```suggestion
    : SimpleBenchmarkReporter(title), exponents_(exponents) {
```

Reply:
I'm afraid we do not have matching constructor for SimpleBenchmarkReporter. Should I add one and apply this suggestion?

Reply:
Oh! you're right. that would be better!